picard/debug_opts.py

```diff
@@ -81,3 +81,4 @@ def get_registry(cls):
 class DebugOpt(DebugOptEnum):
     WS_REPLIES = 1, N_('Web Service Replies'), N_('Log content of web service replies')
     PLUGIN_FULLPATH = 2, N_('Plugin Fullpath'), N_('Log plugin full paths')
+    WS_POST = 3, N_('Web Service Post Data'), N_('Log data of web service post requests')
```

Comment:
Let's have WS_REPLIES and WS_POST next to each other in the options selection

Reply:
Re-ordered, and rebased.